doc/ref_cert/lfn/chapters/chapter01.md

```diff
@@ -109,9 +110,16 @@ The following best practices have been adopted to ensure verification and valida
 <a name="1.6"></a>
 ## 1.6 Verification methodologies
 
-Perform VNF interoperability verifications against an implementation of CNTT reference architecture, leveraging existing OPNFV Intake Process. Upstream projects will define features/capabilities, test scenarios, and test cases to augment existing OVP test harnesses to be executed via the OVP Ecosystem.
+Prior to onboarding a VNF the NFVI must be tested in Network, Compute and Storage domains to ensure that it provides required capabilities that allow the VNF to behave as expected.
+
+Predicting networking performance may be particularly difficult to establish and can be extremely sensitive to configuration details of the NFVi. Testing should be from the perspective of the VNF and create an environment that represents a realistic NFVi setup using open-source test tools and test workloads (i.e. test VNFs) that mimic how a real VNF will consume the NFVi. At this stage of testing we assume that the NFVi vendor has fully debugged and optimized the NFVI components and stack. With this approach the above objectives can be achieved:
+
+1. VNF vendors will know the capabilities of the NFVi and be able to forecast performance of the VNF. By knowing how the NFVI has been tested (and having the open-source test tools including workloads) they can reproduce test results or produce their own results using their own NFVI configurations.
```

Comment:
@trevgc the intent is not to have VNFs vendors to test using their own NFVI configuration. i recommend removing this sentence.

Reply:
+1.